archetypes/jbehave-guice-archetype/src/main/resources/archetype-resources/src/main/java/MyStories.java

```diff
@@ -65,11 +67,13 @@ public Configuration configuration() {
         LoadFromClasspath resourceLoader = new LoadFromClasspath(embeddableClass);
         TableTransformers tableTransformers = new TableTransformers();
         ParameterControls parameterControls = new ParameterControls();
+        // TableTransformerMonitor default instance. Other implementation may be used
+        TableTransformerMonitor transformerMonitor = new NullTableTransformerMonitor();
```

Comment:
please avoid adding `NullTableTransformerMonitor` as it doesn't bring any value here, you can just overload constructors of `ExamplesTableFactory`

Reply:
here and below